src/pages/workspaces/workspace/workflows/LaunchAnalysisModal.js

```diff
@@ -80,11 +68,9 @@ const LaunchAnalysisModal = ({
   const entityCount = Utils.cond(
     [processSingle, () => 1],
     [type === processSnapshotTable, () => entityMetadata[rootEntityType].count],
-    [type === chooseRows || type === chooseSets, () => _.size(selectedEntities)],
-    [type === processAll, () => entityMetadata[rootEntityType].count],
-    [type === processAllAsSet, () => 1],
-    [type === chooseSetComponents, () => 1],
-    [type === processMergedSet, () => _.flow(mergeSets, _.uniqBy('entityName'))(selectedEntities).length]
+    [type === chooseRootType, () => _.size(selectedEntities)],
+    [type === chooseBaseType, () => 1],
+    [type === chooseSetType, () => _.flow(mergeSets, _.uniqBy('entityName'))(selectedEntities).length]
```

Comment:
does this `cond` need a base case?

Reply:
A default? It shouldn't, since we're covering all available options.